tools/cli/flags.go

```diff
@@ -213,6 +215,8 @@ const (
 	FlagMaxMessageCountWithAlias          = FlagMaxMessageCount + ", mmc"
 	FlagLastMessageID                     = "last_message_id"
 	FlagLastMessageIDWithAlias            = FlagLastMessageID + ", lm"
+	FlagGoRoutineCount                    = "go_routine_count"
```

Comment:
nit: Probably `FlagCurrency`? I don't think we need to reveal the fact that CLI is implemented with Golang.